src/styles/blocks/card.scss

```diff
@@ -0,0 +1,117 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: $main-text-color;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  background: url(../images/imac.jpeg);
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  background-size: 160px 134px;
+  background-position: top 32px center;
+  padding: 206px 16px 16px;
+  width: 200px;
+
+  &__name {
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin-bottom: 24px;
+  }
+
+  &__stars {
+    display: flex;
+
+    &-point {
+      background: url(../images/star.svg);
+      background-repeat: no-repeat;
+      background-position: center;
+      width: 16px;
+      height: 16px;
+    }
+
+    &-point:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &__stars > &__stars-point:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
move it to another file as a separate block

Reply:
done